unit_tests/test_nemesis.py

```diff
@@ -12,9 +13,11 @@
 from sdcm.cluster_gce import ScyllaGCECluster
 from sdcm.cluster_aws import ScyllaAWSCluster
 from sdcm.cluster_docker import ScyllaDockerCluster
+from .test_tester import ClusterTesterForTests
```

Comment:
Do not use relative imports. It may easily become a problematic point.

Reply:
fixed